Sources/Jobs/Middleware/MetricsMiddleware.swift

```diff
@@ -0,0 +1,227 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2025 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import Metrics
+
+public struct MetricsJobMiddleware: JobMiddleware {
+    public init() {}
+
+    /// Counter label
+    @usableFromInline
+    static let counterLabel: String = "swift.jobs"
+    /// Job duration timer label
+    @usableFromInline
+    static let timerLabel: String = "swift.jobs.duration"
+    /// Job queued timer label
+    @usableFromInline
+    static let queuedTimerLabel: String = "swift.jobs.queued.duration"
+    /// Meter label for Processsing, Queued
+    @usableFromInline
+    static let meterLabel: String = "swift.jobs.meter"
+    /// Counter label for discarded jobs
+    @usableFromInline
+    static let discardedCounter: String = "swift.jobs.discarded"
```

Comment:
Shouldn't it be labeled `jobs.swift` since it's swift-jobs? Just like it'd be `com.apple.my-service` rather than `apple.com.my-service`

Reply:
These are the metrics we already had